node/external/transactionAPI/gasUsedAndFeeProcessor.go

```diff
@@ -49,6 +93,101 @@ func (gfp *gasUsedAndFeeProcessor) computeAndAttachGasUsedAndFee(tx *transaction
 	gfp.prepareTxWithResultsBasedOnLogs(tx, hasRefundForSender)
 }
 
+func (gfp *gasUsedAndFeeProcessor) getFeeOfRelayed(tx *transaction.ApiTransactionResult) (*big.Int, bool) {
+	if !tx.IsRelayed {
+		return nil, false
+	}
+
+	if len(tx.InnerTransactions) > 0 {
+		return gfp.feeComputer.ComputeTransactionFee(tx), true
+	}
+
+	if len(tx.Data) == 0 {
+		return nil, false
+	}
+
+	funcName, args, err := gfp.argsParser.ParseCallData(string(tx.Data))
+	if err != nil {
+		return nil, false
+	}
+
+	if funcName == core.RelayedTransaction {
+		return gfp.handleRelayedV1(args, tx)
+	}
+
+	if funcName == core.RelayedTransactionV2 {
+		return gfp.handleRelayedV2(args, tx)
+	}
+
+	return nil, false
+}
+
+func (gfp *gasUsedAndFeeProcessor) handleRelayedV1(args [][]byte, tx *transaction.ApiTransactionResult) (*big.Int, bool) {
+	if len(args) != 1 {
+		return nil, false
+	}
+
+	innerTx := &transaction.Transaction{}
+	err := gfp.marshaller.Unmarshal(innerTx, args[0])
+	if err != nil {
+		return nil, false
+	}
+
+	gasUsed := gfp.feeComputer.ComputeGasLimit(tx)
+	fee := gfp.feeComputer.ComputeTxFeeBasedOnGasUsed(tx, gasUsed)
+
+	innerFee := gfp.feeComputer.ComputeTransactionFee(&transaction.ApiTransactionResult{
+		Tx: innerTx,
+	})
+
+	return big.NewInt(0).Add(fee, innerFee), true
+}
+
+func (gfp *gasUsedAndFeeProcessor) handleRelayedV2(args [][]byte, tx *transaction.ApiTransactionResult) (*big.Int, bool) {
+	innerTx := &transaction.Transaction{}
+	innerTx.RcvAddr = args[0]
+	innerTx.Nonce = big.NewInt(0).SetBytes(args[1]).Uint64()
+	innerTx.Data = args[2]
+	innerTx.Signature = args[3]
+	innerTx.Value = big.NewInt(0)
+	innerTx.GasPrice = tx.GasPrice
+	innerTx.GasLimit = tx.GasLimit - gfp.feeComputer.ComputeGasLimit(tx)
+	innerTx.SndAddr = tx.Tx.GetRcvAddr()
+
+	gasUsed := gfp.feeComputer.ComputeGasLimit(tx)
+	fee := gfp.feeComputer.ComputeTxFeeBasedOnGasUsed(tx, gasUsed)
+
+	innerFee := gfp.feeComputer.ComputeTransactionFee(&transaction.ApiTransactionResult{
+		Tx: innerTx,
+	})
+
+	return big.NewInt(0).Add(fee, innerFee), true
+}
+
+func (gfp *gasUsedAndFeeProcessor) getGuardianOperationCost(tx *transaction.ApiTransactionResult) uint64 {
+	gasSchedule, err := gfp.gasScheduleNotifier.GasScheduleForEpoch(tx.Epoch)
+	if err != nil {
+		return 0
+	}
+
+	switch tx.Operation {
+	case core.BuiltInFunctionSetGuardian:
```

Comment:
Related to the above comment: processing flow might be extended in the future, with additional built-in functions, and we'll forget to list them here, as well.

Reply:
At least found in two places:
 - node/external/transactionAPI/gasUsedAndFeeProcessor.go
 - outport/process/transactionsfee/transactionsFeeProcessor.go